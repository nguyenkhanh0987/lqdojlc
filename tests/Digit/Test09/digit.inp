100
647965796 783490469
994267525 996855766
569092342 999362710
524337392 548475613
57957108 278845810
160133000 842360768
931275312 960632356
49877851 273294267
137293573 210022130
616052832 733702067
95648796 573079605
26247470 56471835
38581104 413378359
114607695 632656610
317202913 466131022
786257286 852757868
89333671 604575846
800281474 945383485
165230231 356112107
308337079 359653977
450819313 579136191
541114378 742740218
581108114 865629706
191996937 252830099
443296524 542081478
618293109 805717853
978575127 991460140
818749469 880609798
178376475 330344522
656354702 808635225
638329179 660474342
955910856 973479249
758639661 773543283
782166692 933449637
589725869 988769486
119353058 617480158
465316636 990514037
866946682 953604061
909640375 962301882
194678001 919219633
904854028 975775964
524466950 984067439
473963421 887809878
418084067 600124946
226305176 407694072
782458495 844358145
407989138 594281199
292169698 646432582
767827926 956259676
659216465 696918313
318168720 397357116
525975384 551825718
579545921 820143843
186013214 246965770
618502407 866481914
855974478 941611592
318810337 500644765
124957704 488812359
147115424 289190353
183329700 636106137
200877021 653591905
429757832 822939833
212395702 973122455
38232370 252367583
518076812 693656023
443085035 479979046
794386320 795745044
477343162 773368766
175678909 871750703
565806751 678479714
673127925 722569525
264369036 503396542
470848642 729613732
799534836 923764362
685761358 801963465
209034175 426633881
239529925 536233890
254834213 948604578
679949741 752047573
225649400 306025731
215708536 491562112
586172122 852690674
149664274 661805978
287161762 998232785
139048829 475744969
573567932 914426079
635434492 831900631
591012147 749471519
595711734 850960189
840242421 933980205
181072895 883803261
343766520 501052096
211218012 532051156
544068414 775624863
25148663 713195475
104725278 710197278
150176601 603613252
39489847 536609094
378281502 432270406
665783315 770270310
