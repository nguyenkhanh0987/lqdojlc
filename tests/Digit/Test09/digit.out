13538125
286637
37621708
2104820
42310423
88700833
1413700
43140032
15013733
10916670
73794379
5893341
68067517
77385026
21082968
4224513
80783303
8665790
32474702
6210164
16891176
21254611
25780314
9029913
12812398
18232602
1380558
4212753
25673424
15181594
2049693
1306931
1735000
10724633
32909350
76017501
49449805
5673548
2234234
86044296
3387557
40794067
41549617
24210346
30403781
4285868
23995965
45738970
13648098
5794403
13427604
2418301
23287729
9512715
21446803
5889125
27546593
59967408
26945798
64624697
61725657
42882411
86386054
41144212
20097668
5517293
223749
31631822
87168337
12491604
5094839
37507595
29058080
7880280
11044854
33972869
44362026
78524259
6233745
15084704
42935802
23636438
74798541
76052104
55531128
31140286
17928465
14378839
21590453
6821382
87420740
24526232
47412330
24036632
105478007
87246373
70488519
83137640
7460087
9603993
